permissions/src/iosMain/kotlin/dev/icerock/moko/permissions/ios/MotionPermissionDelegate.kt

```diff
@@ -0,0 +1,30 @@
+package dev.icerock.moko.permissions.ios
+
+import dev.icerock.moko.permissions.PermissionState
+import platform.CoreMotion.CMAuthorizationStatusAuthorized
+import platform.CoreMotion.CMAuthorizationStatusDenied
+import platform.CoreMotion.CMAuthorizationStatusNotDetermined
+import platform.CoreMotion.CMAuthorizationStatusRestricted
+import platform.CoreMotion.CMMotionActivityManager
+import platform.Foundation.NSOperationQueue
+
+internal class MotionPermissionDelegate : PermissionDelegate {
+    override suspend fun providePermission() {
+        val manager = CMMotionActivityManager()
+        manager.startActivityUpdatesToQueue(NSOperationQueue.mainQueue) {}
```

Comment:
is it tested? we can request permission only by try to start updated receiving?

Reply:
I simplified providing the motion permission on iOS. This is the official way to request motion permission on iOS. We request it by when we use it for the first time, so we perform a single collection.